### PR TITLE
Add the ability to ignore certain methods

### PR DESCRIPTION
**Pull request by grantholle**
This allows you to exclude certain methods from being tracked in page views, i.e. limiting the tracking to just GET requests.

### CODE REVIEW

File: config/analytics.php [hunk 1]
```diff
@@ -46,6 +46,15 @@
         // '/users/*',
     ],
 
+    /**
+     * Ignore methods.
+     *
+     * The HTTP verbs/methods that should be excluded from page view tracking.
+     */
+    'ignoreMethods' => [
```
Comment (tomirons) [inline config/analytics.php:54]: Can you change this to use `snake_case`?

**Review by tomirons — CHANGES_REQUESTED** [review]

File: config/analytics.php (hunk 1 above)
Comment (grantholle) [reply config/analytics.php:54]: I also prefer snake case, but opted for camel case due to the existence of `ignoredIPs` and `ignoreRobots`.

Unless there's a good reason to break the current convention.

Comment (andreaselia) [reply config/analytics.php:54]: That's probably my fault, sorry. I also prefer snake case to camel case, but would rather have consistency here and not introduce any breaking changes. Thanks both 👍

**Comment by andreaselia** [comment]
@grantholle just a minor merge conflict and then this is good to go mate 👍

**Comment by grantholle** [comment]
Good to go, thanks!

**Comment by andreaselia** [comment]
Thank you, Grant! 🚀

**Pull request merged**